dom0/sd-proxy.sls

```diff
@@ -10,32 +10,33 @@
 ##
 
 include:
-  - qvm.template-whonix-ws
   - sd-whonix
 
 sd-proxy-template:
   qvm.vm:
-    - name: sd-proxy-template
+    - name: sd-proxy-buster-template
     - clone:
-      - source: whonix-ws-14
+      - source: whonix-ws-15
       - label: blue
     - tags:
       - add:
         - sd-workstation
+        - sd-buster
```

Comment:
I think we must require sls sd-upgrade-templates here , to ensure the template update was successful before cloning whonix-ws-15 to sd-proxy-buster-template